_posts/2013-05-02-install.markdown

```diff
@@ -342,5 +360,8 @@ Go to [https://nitrous.io](https://nitrous.io/) and signup.
 * Everything you need is right here in you browser window - you do not need to start an editor or terminal anywhere else
 * If you are following a guide or tutorial, use the commands for Linux even if you are on a Windows computer - your operating system does not matter, since all commands are run on your project's machine in the cloud, which is running Linux
 * If a guide or tutorial asks you to start the rails server with 'rails server' or 'rails s', append '-b 0.0.0.0' to this command so that you run 'rails server -b 0.0.0.0'
```

Comment:
This guide usually formats terminal commands using backticks. For consistency, I suggest to use backticks instead of single quotes for rails server, rails s etc in the previous list item.